p2p/raintree/network.go

```diff
@@ -78,8 +103,21 @@ func (n *rainTreeNetwork) networkBroadcastAtLevel(data []byte, level uint32, non
 	return nil
 }
 
+// Cleanup layer is just another send left / right
+// This layer happens at level -1
+func (n *rainTreeNetwork) cleanupLayer() []target {
+	// TODO (Team) unhappy path where the left / right nodes are down
+	// 			   (continue to search left and right until you have a hit)
+	return n.getCleanupTargets()
+}
+
+// Redundancy layer is simply a redundant send to the original +1/3 && -1/3
+func (n *rainTreeNetwork) redundancyLayer() (level int32, msgLevel int32) {
+	return int32(n.peersManager.maxNumLevels), 0
+}
```

Comment:
I might be misunderstanding something because from https://github.com/pokt-network/pocket-network-protocol/tree/main/p2p I gather that redundancy consists on sending messages from the originator to ALL the nodes at level 0

Reply:
https://github.com/pokt-network/pocket/pull/368#discussion_r1037760641